docs/src/examples/components/Reaction/Types/ReactionExample.shorthand.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react'
+import { Reaction, Flex } from '@stardust-ui/react'
+
+const ReactionExample = () => (
+  <Flex gap="gap.small">
```

Comment:
I'm normalizing the spacing between items to 8px using `gap="gap.smaller"`, see: https://github.com/stardust-ui/react/pull/945/files#r260245496

can we do the same here?

Reply:
I added the ReactionGroup component, will get rid of this one.